packages/@okta/vuepress-theme-prose/components/SidebarItem.vue

```diff
@@ -1,9 +1,7 @@
 <template>
-  <li :class="{ subnav: link.subLinks, hidden: hidden }">
-    <div class="link-wrap">
-
-      <div v-if="entityType === types.link">
-        <router-link
+  <li :class="{'link-wrap': true, 'subnav-active': link.iHaveChildrenActive, hidden: hidden }">
```

Comment:
can leverage sibling selector instead?
e.g. `  .tree-nav-link.children-active + .sections {`

Reply:
We can't do that, cos we have list of siblings, where any of siblings can be expanded. Also, they are pre-rendered all together, so, there only one chance of adding styles - programmatically  adding them to <li>, either inline css or additional style. I'm not a big fan of inline styles adding, I think they are appropriate only in several cases.